### DISCUSSION TITLE
Welcome to OneDrive-Index Discussions!

### DISCUSSION THREAD

**Discussion by iRedScarf** [iRedScarf/onedrive-index#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This project is going to be the future of [onedrive-vercel-index](https://github.com/iRedScarf/onedrive-vercel-index). I will post the ongoing status of this project here.

> This project is relatively complete, and I just plan to make some small optimizations next.

If you are running into trouble on deployments, please read the [documentation](https://ovi.swo.moe/docs/getting-started) written by the original author [@spencerwooo](https://github.com/spencerwooo) first. **If you must ask a question, please use discussions for your problems during deployments and builds.** Issues are for bugs and bugs only.

**Comment by hirockstar** [comment]
if i try to update ui to next level is that okey to do on your project as Contributors?? 
Here is small demo 
[RDRIVE](https://rdrive.org)

**Reply by iRedScarf** [reply]
Your demo is very nice, I like it. I also have the idea of ​​​​modifying the UI, but it requires a lot of file and code modification, so I feel a little timid.

**Reply by hirockstar** [reply]
No worries I will try to do whenever I'm free

**Reply by iRedScarf** [reply]
wow that's great

**Reply by hirockstar** [reply]
bro give us permission so we can make pull request
